### PR TITLE
Add resources/matt davis

### PR DESCRIPTION
**Pull request by TeddyGavi**
Hello! Please review these added resources and formatting changes. Many of the formatting changes have been applied by prettier. It appears that there was some unused * in front of a number of list items. If these are required and/or you don't agree with the formatting updates I will refactor and set prettier to ignore the document. Turns out prettier doesn't play all that well with *.md files. :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,79 +12,88 @@ Country-specific resources are marked with a flag and country icons, like :trian
...
-|                                   | [4 Day Week](https://4dayweek.io/)                                      | 4-day work week    |
+<!-- prettier-ignore-start -->
 
+#### :point_right: Discord server invites expire in 7 days, so no server invites are provided. The following contain links to starting points. Discover on your own! :arrow_heading_down: If the community _does not_ have a Discord Server it will be marked with a :no_entry:
 
-## :money_with_wings:Negotiating the offer
+<!-- prettier-ignore-end -->
+
+- **100Devs** :dart: Leon Noel, Managing Director of Engineering @ [Resilient Coders](https://resilientcoders.org/) helps you become a software engineer for free: :point_right: [link](https://leonnoel.com/100devs/) :rainbow:
+- **#DearJuniorDev** Follow [Jessica Parker](https://www.linkedin.com/in/devjessparker/) on LinkedIn.
```
Comment (dariatsvetkova) [inline README.md]: I love these suggestions! In terms of classification, maybe this one belongs in a separate category, something like `Content` or `Influencers`?

**Comment by dariatsvetkova** [comment]
> Hello! Please review these added resources and formatting changes. Many of the formatting changes have been applied by prettier. It appears that there was some unused * in front of a number of list items. If these are required and/or you don't agree with the formatting updates I will refactor and set prettier to ignore the document. Turns out prettier doesn't play all that well with *.md files. 😄

I haven't applied prettier to this file before because I usually edit it in the browser 😄 But at the end of the day, `*` and `-` do the same thing in markdown, so you are welcome to use whichever works best with your code formatter.

File: README.md (hunk 1 above)
Comment (TeddyGavi) [reply README.md]: Thank you! I moved DearJuniorDev into its own category `Social Content`. that does make more sense despite there being a discord community as well, its primarily through LinkedIn and in the future more social resources can be added.

**Review by dariatsvetkova — APPROVED** [review]
Thank you for contributing and adding some great resources! 🙏

**Pull request merged**